### ISSUE TITLE
Accordion box pixel polishes - Variability Screen

### ISSUE THREAD

**Issue by catherinecarter** [phetsims/center-and-variability#206]
This issue is to identify pixel polishes within the accordion box on the Variability screen:

- [x] The font sizes are different at the top of each measure of spread. If possible, can we make them all match what's on the Range scene?
- [x] The icons for the Range, IQR, and MAD bounce left/right when radio buttons are toggled (the icons within the accordion box on the right). Let's put them in the same place. So whatever is longest (I think it's Range), please match the location of the IQR and MAD icons with the position of the Range icon.
- [ ] Brackets above the rectangles for the measures of spread are not consistent in height. Let's make them match each other.  UPDATE: See comment below
- [x] The number line on the boxplot is slightly different height (it's lower) than the other two scenes.
- [ ] The arrow in the icons mentioned above need to be a bit bigger. The arrow could match the one inside the Interval Tool icon - let's see how that looks since the rectangle holding the arrows within the accordion box are a little bigger than the Interval tool icon.
- [x] The measures of center and spread on the left side of the accordion box need to come up a little bit higher from the bottom
- [x] The spacing between measures of spread on the left is inconsistent. The median/IQR could be slightly decreased and mean/MAD could be increased so their location and spacing match each other.
- [x] When there's one data point or all the points are stacked in the same location, the MAD says, "MAD = ?" and should say, "MAD = 0" since the MAD of a single point is zero, and the MAD of a stack is also zero.
- [x] Same comment for the IQR. When the IQR is zero, the measure on the left is appearing as, "IQR = ?" when it should say, "IQR = 0"
- [ ] The MAD horizontal lines are going into and under the number line when the number of data points is increased. 
- [x] On the IQR radio button, the phrase "Need at least 5 kicks" is overlapping the number line. This needs to be raised into the graph area.
- [ ] For the IQR and MAD, when these values are zero, there's a zero appearing above the stack of balls. On the Range, there's no value of 0 showing, but the value, "Range = 0" is visible on the left. When the IQR and MAD are zero, I believe we decided there won't be a value of zero showing above the stack, but maybe I'm just thinking we made that decision for the Range and not the others? Either way, I think showing nothing above the stack is a reasonable way to show the measure of spread = 0 (but with the value showing on the left).
- [x] The color of dots/x's is dark enough that it's sometimes hard to parse the graphs. Let's try #b2b2b2 and see if these are too light or just right to see the graphs.
- [ ] Reduce the notch_height from 10 pixels to 6

It's a lot in one post.

**Comment by samreid** [comment]
> The font sizes are different at the top of each measure of spread. If possible, can we make them all match what's on the Range scene?

I factored this out, but was a bit confused because all 3 already had `new PhetFont(13)` so they should have matched.  Unless something changed between when you wrote this and now.

**Comment by catherinecarter** [comment]
I wrote: 
> Brackets above the rectangles for the measures of spread are not consistent in height. Let's make them match each other.

You replied: 
> It looks like they are all using MedianBarNode.NOTCH_HEIGHT which is 10. I'm having difficulty seeing the difference in them and something else may be happening.

I don't think I described what I meant well. I meant that their height relative to the accordion box, like the distance from the top, is different. Hopefully that clarifies a bit.

Also, I'm wondering if we can make the distance from the top of the rectangle to the top of the measurements bar stay the same, but reduce the length of the vertical pieces stemming from the measurement bar down? I believe it might be the notch_height, but I'm not sure. If this is the correct description, I believe this might be set to 10? Can we change it to 6 and see how it looks? Thanks.

Putting it as a checkbox in the list above.

**Comment by matthew-blackman** [comment]
@marlitas @samreid and I discussed this, and would like to defer working on it until all dynamic string work is complete.

**Comment by matthew-blackman** [comment]
@catherinecarter the above commits adjust the size of the IQR/range/MAD bar 'notches' and the space between the accordion box title and the IQRNode. Ready for you to review!

**Comment by catherinecarter** [comment]
Notch sizes look great! The space between the box title and the IQRNode also look good. The MAD and IQR font sizes for their measure look different (range and IQR appear to match, MAD is bigger). 

The heights of all three rectangles look to be ending in different spots, and the distance from the rectangle to the notches look different, too. Not a priority for interviews, but thought I'd mention it.

**Comment by samreid** [comment]
@catherinecarter mentioned reducing the height of the question bar, and grass size, so we may get more size in the accordion box.  But @catherinecarter could still review this issue to see if there are other aspects to comment on before that part.

**Comment by catherinecarter** [comment]
Closing this giant issue as there are too many things to keep track of. Created new issues for those still open for easier tracking. 
MAD lines below axis: #281
